tests/NumericHelperTest.php

```diff
@@ -37,19 +37,21 @@ public function testToOrdinalWithIncorrectType(): void
     public function normalizeNumberDataProvider(): array
     {
         return [
-
             'French' => ['4 294 967 295,000', '4294967295.000'],
             'German' => ['4 294 967.295,000', '4294967295.000'],
             'Spanish' => ['4.294.967.295,000', '4294967295.000'],
             'English' => ['4,294,967,295.000', '4294967295.000'],
             'Smaller' => ['10,111', '10.111'],
+            [10.0, '10'],
```

Comment:
```suggestion
            'Float' => [10.0, '10'],
```
or something like this

Reply:
Done!